ext/ffi/lib.rs

```diff
@@ -2071,32 +2036,59 @@ unsafe extern "C" fn noop_deleter_callback(
 }
 
 #[op(v8)]
-fn op_ffi_get_buf<FP, 'scope>(
+fn op_ffi_get_ptr_value<'scope>(
+  scope: &mut v8::HandleScope<'scope>,
+  state: &mut deno_core::OpState,
+  ptr: serde_v8::Value<'scope>,
+) -> Result<serde_v8::Value<'scope>, AnyError> {
+  check_unstable(state, "Deno.UnsafePointer#getPointerValue");
+
+  let ptr = if let Ok(value) = v8::Local::<v8::External>::try_from(ptr.v8_value)
+  {
+    value.value() as *const c_char
```

Comment:
The use of `c_char` isn't very self-documenting here?

Reply:
Ah, this should've been `c_void`. Probably VS Code suggestion error.